### PR TITLE
Standard Svelte ordering

### LINKED ISSUE #281: Switch to standard Svelte ordering
Vite handles HMR better with the standard order of `options-scripts-markup-styles`: https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#what-is-the-recommended-node-order-for-svelte-sfc-files

We currently use `options-scripts-styles-markup`.

This is a pretty simple change, but it's going to make for a giant, ugly formatting PR where we touch basically every file.

### PR DESCRIPTION
**Pull request by eyeseast**
Closes #281 

This reformats every Svelte file to use the standard Svelte ordering.

Vite handles HMR better with the standard order of options-scripts-markup-styles: https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#what-is-the-recommended-node-order-for-svelte-sfc-files

If this messes up anybody's local work, it's pretty easy to close this and re-do it later if needed.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -46,30 +46,6 @@ prettier-check:
...
-# Set BROWSER to change the browser. e.g. BROWSER=chromium make browser-test-headful
-browser-test-headful:
-	DEBUG=yes node tests/functional/suites/noindex.js
-
-browser-test-headful-staging:
-	DEBUG=yes node tests/functional/suites/noindex.js --envfile .env.staging
-
-browser-test-debug:
-	DEBUG=yes node debug tests/functional/suites/noindex.js
-
```
Comment (mitchelljkotler) [inline Makefile:72]: Why are these removed?

Comment (eyeseast) [reply Makefile:72]: We're not using them anymore. Browser tests run outside of Docker now, and these refer to files we've already deleted.

Comment (eyeseast) [reply Makefile:72]: Also deleted the `plugins` folder I made for Netlify that we ended up not using.
